packages/edit-site/src/store/actions.js

```diff
@@ -54,7 +54,7 @@ export function* setTemplate( templateId, templateSlug ) {
 	const pageContext = { templateSlug };
 	if ( ! templateSlug ) {
 		const template = yield controls.resolveSelect(
-			'core',
+			coreStore,
```

Comment:
In this file, we should either leave the hardcoded name or use `coreStor.name` because it expects a string.